src/azure-cli/azure/cli/command_modules/network/tests/latest/test_dns_commands.py

```diff
@@ -575,6 +588,10 @@ def test_zone_file_1(self):
             (60, 0, 'issue', 'ca1.contoso.com'),
             (60, 45, 'tag56', 'test test test')
         ])
+        #self._check_naptr(zone, 'mynaptr.' + zn, [
+        #    (3600, 10, 20, 'A', 'EAU+SIP', '', 'domain.com.'),
+        #    (3600, 20, 20, 'U', 'SIP+D2U', '!^(\\+441632960083)$!sip:\\1@example.com!', '.')
+        #])
```

Comment:
why is this commented?

Reply:
In zone1.txt, I have commented out the naptr record types. I will enable them after fixing the zone import logic in another PR.
If I uncomment these lines the unit tests will fail - `DnsParseZoneFiles.test_zone_file_1`

Reply:
I removed this entirely and added a separate zone file